src/components/WorkersTable.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="workers-table">
+    <div class="workers-table__filters">
+      <ResultsCount label="Worker Pool" :count="filteredWorkers.length" class="worker-pools__results" />
+
+      <SearchInput v-model="searchValue" class="worker-pools__search" placeholder="Search worker pools" />
+    </div>
+
+    <p-table :data="filteredWorkers" :columns="columns">
+      <template #name="{ row }">
+        <span>{{ row.name }}</span>
+      </template>
+
+      <template #last-polled="{ row }">
+        <span>{{ formatDateTimeNumeric(row.lastHeartbeatTime) }}</span>
+      </template>
+
+      <template #action-heading>
+        <span />
+      </template>
+
+      <template #action="{ row }">
+        <div class="workers-table__actions">
+          <p-icon-button-menu>
+            <CopyOverflowMenuItem label="Copy ID" :item="row.id" />
+          </p-icon-button-menu>
+        </div>
+      </template>
+
+      <template #empty-state>
+        <PEmptyResults>
+          <template #message>
+            No workers
+          </template>
+          <template v-if="hasFilters" #actions>
+            <p-button size="sm" secondary @click="clear">
+              Clear Filters
+            </p-button>
+          </template>
+        </PEmptyResults>
+      </template>
+    </p-table>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed, ref } from 'vue'
+  import { ResultsCount, SearchInput, CopyOverflowMenuItem } from '@/components'
+  import { useWorkspaceApi } from '@/compositions'
+  import { formatDateTimeNumeric } from '@/utilities/dates'
+
+  const props = defineProps<{
+    workerPoolName: string,
+  }>()
+
+  const api = useWorkspaceApi()
+  const subscriptionOptions = {
+    interval: 30000,
+  }
+
+  const workerPoolNameRef = computed(() => props.workerPoolName)
+
+  const workerPoolWorkersSubscription = useSubscription(api.workerPoolWorkers.getWorkers, [workerPoolNameRef.value], subscriptionOptions)
+  const workerPoolWorkers = computed(() => workerPoolWorkersSubscription.response ?? [])
+
+  const searchValue = ref<string>('')
+
+  const filteredWorkers = computed(() => {
+    if (!searchValue.value) {
+      return workerPoolWorkers.value
+    }
+
+    return workerPoolWorkers.value.filter(key => key.name.toLowerCase().includes(searchValue.value.toLowerCase()),
+    )
+  })
+
+  const hasFilters = computed(() => {
+    return !!searchValue.value
+  })
+
+  const columns = [
+    {
+      property: 'name',
+      label: 'Name',
+    },
+    {
+      label: 'Last Polled',
```

Comment:
not sure if you're aware, certainly falls under nit or 2 ways to accomplish the same thing. But FTR, you can have a property that's different from the label and then use the property in the template

```suggestion
      property: 'lastHeartbeatTime',
      label: 'Last Polled',
```

then in the template 
```html
<template #last-polled="{ value }">
  <span>{{ formatDateTimeNumeric(value) }}</span>
</template>
```